src/commands/createRepo/RepoCreateFromTemplateStep.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Progress } from 'vscode';
+import { AzureWizardExecuteStep } from "vscode-azureextensionui";
+import { ext } from '../../extensionVariables';
+import { RepoData } from '../../gitHubTypings';
+import { createRepoFromTemplate } from "../../utils/gitHubUtils";
+import { localize } from '../../utils/localize';
+import { nonNullProp } from '../../utils/nonNull';
+import { IStaticWebAppWizardContext } from '../createStaticWebApp/IStaticWebAppWizardContext';
+
+export class RepoCreateFromTemplateStep extends AzureWizardExecuteStep<IStaticWebAppWizardContext> {
```

Comment:
I'm a little torn here.  On one hand, this step has a few things different happening when compared to RepoCreateStep, but on the otherhand, a lot of it including the set-up and strings are identical.  

I think I'm okay with either solution, but lean a little bit more towards having one step that has a createRepo and createRepoFromTemplate functions that we can call inside the function.